sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -883,7 +895,7 @@ class Column(protected[sql] val expr: Expression) extends Logging {
    * @group expr_ops
    * @since 1.4.0
    */
-  def as(aliases: Seq[String]): Column = withExpr { MultiAlias(expr, aliases) }
+  def as(aliases: Seq[String]): Column = name(aliases)
```

Comment:
I see, I will revert three back, include def as(alias: String, metadata: Metadata): Column. Let me know it is ok. Thanks.